packages/lib-utils/src/k8s/k8s-utils.ts

```diff
@@ -15,14 +15,36 @@ import type {
 import type { WebSocketOptions } from '../web-socket/types';
 import { WebSocketFactory } from '../web-socket/WebSocketFactory';
 
+const ACTIVE_WORKSPACE_KEY = 'sdk/active-workspace';
+
+/**
+ * @returns the activeWorkspace as a string or null
+ */
+export function getActiveWorkspace() {
+  return localStorage.getItem(ACTIVE_WORKSPACE_KEY);
+}
+
+/**
+ * @param workspace - the string name of the workspace you wish to set as active
+ */
+export function setActiveWorkspace(workspace: string) {
+  localStorage.setItem(ACTIVE_WORKSPACE_KEY, workspace);
+}
+
 const getQueryString = (queryParams: QueryParams) =>
   Object.entries(queryParams)
     .map(([key, value = '']) => `${encodeURIComponent(key)}=${encodeURIComponent(value)}`)
     .join('&');
 
 const getK8sAPIPath = ({ apiGroup = 'core', apiVersion }: K8sModelCommon) => {
+  let path = '';
+  const activeWorkspace = getActiveWorkspace();
```

Comment:
We might need to eventually allow overriding namespace for individual requests (so clients can get resources from different names), but not worry about that now and unblock the team. We can consider as a follow up.

Will anything trigger an update to watch hooks when the workspace changes?

Reply:
@florkbr Sorry, I just edited my comment. I meant _workspaces_, not namespaces. We might have a use case in the future where clients need to get resources from both workspace-a and workspace-b so there's not a single context. We'd need some prop in the request to specify the workspace and fall back to the active workspace when not present.

Also we'll want to make sure the k8s hooks trigger a re-render when the workspace context is updated.

Reply:
@spadgett ah - I see your point now. We actually started down that path already via `pathPrefix`. See: https://github.com/openshift/dynamic-plugin-sdk/blob/main/packages/lib-utils/src/k8s/hooks/useK8sWatchResource.ts#L36

The idea would be a plugin could use the hooks with different `pathPrefix` set to achieve fetching resources from multiple workspaces at once. However there are a few challenges:
1. There is a bug with `pathPrefix` (more specifically passing any options object to the hook) that causes an infinite rendering loop
2. The hook expects API discovery to load the models. We have not yet updated API discovery to fetch workspace specific resources. We could however update the hook to accept a static model (versus relying on API discovery), but that would push the onus onto the plugin to provide the model. I think we mentioned the workspace model may not change much, so maybe that is ok? 

Finally - we will follow up in another PR to update the hooks to re-render when the active workspace changes (and indicate loading is occurring).

Reply:
@florkbr Sounds good. When using dev sandbox, the resources will be the same regardless of workspace, so we don't need to trigger discovery again.